x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/asset_criticality_privileges.ts

```diff
@@ -145,7 +145,7 @@ export default ({ getService }: FtrProviderContext) => {
     });
 
     // https://github.com/elastic/kibana/issues/183247
-    describe('@serverless @skipInServerlessMKI Asset Criticality Privileges API', () => {
+    describe('@skipInServerless @skipInServerlessMKI Asset Criticality Privileges API', () => {
```

Comment:
`@skipInServerless` is sufficient to exclude this test from both the MKI and PR pipelines. I left `@skipInServerlessMKI` to specify the scope of this exclusion more verbosely.

Reply:
Entity Analytics follow-up issue [here](https://github.com/elastic/kibana/issues/189067).